packages/ragbits-core/src/ragbits/core/llms/base.py

```diff
@@ -130,6 +130,32 @@ async def generate(
 
         return cast(OutputT, response)
 
+    async def generate_streaming(
+        self,
+        prompt: BasePrompt,
+        *,
+        options: LLMOptions | None = None,
+    ) -> AsyncGenerator[str, None]:
+        """
+        Prepares and sends a prompt to the LLM and returns the raw response (without parsing).
```

Comment:
Seems like the documentation copied from `generate_raw` instead of a description referring to this specific method

Reply:
fixed